File: content/en/api/v1/pager-duty-integration/UpdatePagerDutyIntegrationService.java
import com.datadog.api.v1.client.api.PagerDutyIntegrationApi;

public class PagerDutyIntegrationApiExample {

    public static void main(String[] args) {
        PagerDutyIntegrationApi apiInstance = new PagerDutyIntegrationApi();
        String serviceName = serviceName_example; // String | The service name
        PagerDutyServiceKey body = ; // PagerDutyServiceKey | 
        try {
            apiInstance.updatePagerDutyIntegrationService(serviceName, body);
        } catch (ApiException e) {
            System.err.println("Exception when calling PagerDutyIntegrationApi#updatePagerDutyIntegrationService");
            e.printStackTrace();
        }
    }
}

File: content/en/api/v1/pagerduty-integration/UpdatePagerDutyIntegrationService.rs
// Update a single service object returns "OK" response
use datadog_api_client::datadog;
use datadog_api_client::datadogV1::api_pager_duty_integration::PagerDutyIntegrationAPI;
use datadog_api_client::datadogV1::model::PagerDutyServiceKey;

#[tokio::main]
async fn main() {
    let body = PagerDutyServiceKey::new("".to_string());
    let configuration = datadog::Configuration::new();
    let api = PagerDutyIntegrationAPI::with_config(configuration);
    let resp = api
        .update_pager_duty_integration_service("service_name".to_string(), body)
        .await;
    if let Ok(value) = resp {
        println!("{:#?}", value);
    } else {
        println!("{:#?}", resp.unwrap_err());
    }
}
